spec/requests/security_activity_spec.rb

```diff
@@ -210,6 +211,20 @@
       expect_event SecurityActivity::PASSWORD_CHANGED
       expect_event_on_security_page SecurityActivity::PASSWORD_CHANGED
     end
+
+    it "record a USER_DESTROYED event" do
+      attribute_service_url = 'example.gov.uk'
+
+      ClimateControl.modify ATTRIBUTE_SERVICE_URL: attribute_service_url do
+        allow(Rails.logger).to receive(:public_send)
+        stub_request(:delete, "#{attribute_service_url}/v1/attributes/all")
+          .with(headers: { accept: "application/json", authorization: "Bearer #{bearer_token}" })
```

Comment:
I suspect you can omit this `.with` part and webmock will just not care about the headers in that case

Reply:
ace, good shout have removed that and done a lint